src/rules/selector-no-attribute/__tests__/index.js

```diff
@@ -0,0 +1,15 @@
+import { ruleTester } from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(null, tr => {
+  tr.ok("foo {}")
+  tr.ok(".bar {}")
+  tr.ok("foo .bar {}")
+
+  tr.notOk("[foo] {}", messages.rejected)
+  tr.notOk("a[rel=\"external\"] {}", messages.rejected)
+  tr.notOk("a, .foo[type=\"text\"] {}", messages.rejected)
+  tr.notOk("a > [foo] {}", messages.rejected)
```

Comment:
Can we add a single quotation marks reject test please?

`tr.notOk("a[rel='external'] {}", messages.rejected)`